commit c45c00eed4750f6b28ef14af0122922224607f41
Author: Daniel Hofstetter <daniel.hofstetter@42dh.com>
Date:   Wed Mar 6 17:51:19 2024 +0100

    ls: use chrono::TimeDelta::try_seconds instead of
    
    deprecated chrono::TimeDelta::seconds

Index: coreutils/src/uu/ls/src/ls.rs
===================================================================
--- coreutils.orig/src/uu/ls/src/ls.rs
+++ coreutils/src/uu/ls/src/ls.rs
@@ -2981,8 +2981,7 @@ fn display_date(metadata: &Metadata, con
         Some(time) => {
             //Date is recent if from past 6 months
             //According to GNU a Gregorian year has 365.2425 * 24 * 60 * 60 == 31556952 seconds on the average.
-            let recent = time + chrono::TimeDelta::try_seconds(31_556_952 / 2).unwrap()
-                > chrono::Local::now();
+            let recent = time + chrono::TimeDelta::seconds(31_556_952 / 2) > chrono::Local::now();
 
             match &config.time_style {
                 TimeStyle::FullIso => time.format("%Y-%m-%d %H:%M:%S.%f %z"),
